package/cluster/network/definition.yaml

```diff
@@ -21,6 +21,46 @@ spec:
                 id:
                   type: string
                   description: ID of this Network that other objects will use to refer to it.
+                region:
```

Comment:
The field path here is `spec.region`, where in other XRDs it is `spec.parameters.region`. I know the `id` field is from the first versions of the XRD, but it seems that we should be consistent across definitions in order to reduce errors.

Reply:
moved to spec.parameters.region.